packages/react/src/auth/useSendSignInLinkToEmailMutation.ts

```diff
@@ -0,0 +1,29 @@
+import { useMutation, type UseMutationOptions } from "@tanstack/react-query";
+import {
+  type ActionCodeSettings,
+  type Auth,
+  AuthError,
+  sendSignInLinkToEmail,
+} from "firebase/auth";
+
+type SendSignInLinkParams = {
+  email: string;
+  actionCodeSettings: ActionCodeSettings;
```

Comment:
Should this be optional?

Reply:
the [actionCodeSettings](https://firebase.google.com/docs/reference/js/auth.actioncodesettings.md#actioncodesettings_interface) is required because it contains [url](https://firebase.google.com/docs/reference/js/auth.actioncodesettings.md#actioncodesettingsurl) to which the sign-in link redirects, the [handleCodeInApp](https://firebase.google.com/docs/reference/js/auth.actioncodesettings.md#actioncodesettingshandlecodeinapp) which is required as well. it is the [iOS](https://firebase.google.com/docs/reference/js/auth.actioncodesettings.md#actioncodesettingsios) and [android](https://firebase.google.com/docs/reference/js/auth.actioncodesettings.md#actioncodesettingsandroid) configurations that are optional within the `actionCodeSettings`

so i think `actionCodeSettings` shouldnt be optional